### DISCUSSION TITLE
Crypto scamming on Github

### DISCUSSION THREAD

**Discussion by markohaapala** [community/community#141564] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

My friend got scammed by a transient user who redirected him to a scam site (thedappsupport.(live/com)). He lost ~600eur worth of cryptos. Does github have a blacklist where to report these nefarious sites, so they can be removed from even being typed into the comments?

Regards,

Marko

**Comment by ebndev** [comment, marked as answer]
Hi @markohaapala, 👋🏻 The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by markohaapala** [reply]
Thank you for the info. This can be closed.
